source/common/http/conn_manager_impl.h

```diff
@@ -294,7 +317,8 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     void addStreamEncoderFilterWorker(StreamEncoderFilterSharedPtr filter, bool dual_filter);
     void chargeStats(const HeaderMap& headers);
     std::list<ActiveStreamEncoderFilterPtr>::iterator
-    commonEncodePrefix(ActiveStreamEncoderFilter* filter, bool end_stream);
+    commonEncodePrefix(ActiveStreamEncoderFilter* filter, bool end_stream,
```

Comment:
Can you add comments here on what `check_iteration_state` is doing as a param?

Reply:
Thanks for catching that! check_iteration_state was to indicate if we should check StopAll state for the current filter. But since the latest code resets iteration state before iterating filters, I removed the unnecessary variable.